### DISCUSSION TITLE
为什么不支持Lingva了呢？

### DISCUSSION THREAD

**Discussion by caecfd** [pot-app/pot-desktop#448] [General]
谷歌引擎经常挂掉，之前版本的Lingva很好用的，怎么到2.0就没了呢？强烈建议把Lingva加回来啊

**Comment by Pylogmon** [comment]
可以装插件 https://github.com/pot-app/pot-app-translate-plugin-template

**Comment by caecfd** [comment]
您好，已收到您的邮件，我会在最短的时间内给您回复，谢谢！
